vonk/releasenotes/releasenotes.rst

```diff
@@ -17,6 +17,53 @@ Upgrading Vonk
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Vonk.
 
+.. _vonk_releasenotes_350:
```

Comment:
General remark: perhaps move some older notes (<3.0) to the 'previous release notes'?